### ISSUE TITLE
Why `softmax` was not implemented in the last layer to classify?

### ISSUE THREAD

**Issue by zugexiaodui** [cypw/PyTorch-MFNet#31]
I didn't find that you use 'softmax' after the last layer(after 'classifier' in 'mfnet_3d.py') or in 'model.fit' (in 'model.py'), but you still use 'CrossEntropyLoss'. However,  in 'evaluate_video_ucf101_split1.py' I found 'softmax' was used. Did I overlook anything? If you really did not use 'softmax' in training process, why did you do so? Thanks!

**Comment by cypw** [comment]
I adopt [torch.nn.CrossEntropyLoss](https://pytorch.org/docs/master/nn.html#torch.nn.CrossEntropyLoss) during trainnig [link](https://github.com/cypw/PyTorch-MFNet/blob/master/train_model.py#L39).
*This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class.

An extra [torch.nn.Softmax](https://github.com/cypw/PyTorch-MFNet/blob/master/test/evaluate_video_ucf101_split1.py#L149) is added during testing to normalize the outputs as a probability distribution, i.e. sum=1.

**Comment by zugexiaodui** [comment]
> I adopt [torch.nn.CrossEntropyLoss](https://pytorch.org/docs/master/nn.html#torch.nn.CrossEntropyLoss) during trainnig [link](https://github.com/cypw/PyTorch-MFNet/blob/master/train_model.py#L39).
> *This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class.
> 
> An extra [torch.nn.Softmax](https://github.com/cypw/PyTorch-MFNet/blob/master/test/evaluate_video_ucf101_split1.py#L149) is added during testing to normalize the outputs as a probability distribution, i.e. sum=1.

I get it ! Thanks!
